giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/aspects/access/RestSecurityAspect.java

```diff
@@ -77,6 +78,9 @@
     
     @Autowired
     private List<IChecker> checkers;
+
+    @Autowired
+    private GilesTokenConfig tokenConfig;
```

Comment:
and since the message handler logs error, they shouldn't be logged before the handler is called.